src/main/java/DBconnect/ConnectionPool.java

```diff
@@ -0,0 +1,51 @@
+package DBconnect;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class ConnectionPool {
+
+	public Connection getConnection() {
+		
+		try {
+			Class.forName("com.mysql.jdbc.Driver");
+		} catch (ClassNotFoundException e) {
+			System.out.println("MySQL JDBC Driver Not found");
+			e.printStackTrace();
+			return null;
+		}
+
+		Connection connection = null;
+
+		try {
+			connection = DriverManager.getConnection(
+					"jdbc:mysql://localhost:3306/visualizer", "root", "");
```

Comment:
@tharindu99 pls change database details(i.e URL, username, passwd) to read from a configuration file